cmd/helpMessages.go

```diff
@@ -292,9 +292,49 @@ Sync an entire directory but exclude certain files from the scope (ex: every fil
 Note: if include/exclude flags are used together, only files matching the include patterns would be looked at, but those matching the exclude patterns would be always be ignored.
 `
 
+// ===================================== DOC COMMAND ===================================== //
+
 const docCmdShortDescription = "Generates documentation for the tool in Markdown format"
 
 const docCmdLongDescription = `Generates documentation for the tool in Markdown format, and stores them in the designated location.
 
 By default, the files are stored in a folder named 'doc' inside the current directory.
 `
+
+// ===================================== BENCH COMMAND ===================================== //
+const benchCmdShortDescription = "Performs a performance benchmark"
+
+// TODO: document whether we delete the uploaded data
+
+const benchCmdLongDescription = `
+Performs a performance benchmark by generating test data and uploading it to a specified destination.
```

Comment:
>Performs [](start = 0, length = 8)

Minor: change wording to be less redundant.

Reply:
Thanks. Done.